--- simage/CMakeLists.txt.orig	2019-12-24 02:00:09.000000000 +0100
+++ simage/CMakeLists.txt	2020-11-30 19:57:02.597525377 +0100
@@ -432,7 +432,7 @@
 
 if(GIF_FOUND)
   target_include_directories(simage PRIVATE ${GIF_INCLUDE_DIR})
-  target_link_libraries(simage PRIVATE ${GIF_LIBRARIES})
+  target_link_libraries(simage PRIVATE ${GIF_LIBRARIES} gifutil)
 endif()
 
 if(JASPER_FOUND)
--- simage/src/simage_gif.c.orig	2019-12-24 02:00:09.000000000 +0100
+++ simage/src/simage_gif.c	2020-11-30 19:57:31.320703103 +0100
@@ -30,6 +30,7 @@
 #include <string.h>
 #include <stdio.h>
 #include <gif_lib.h>
+#include <gif_util.h>
 
 #if GIFLIB_MAJOR > 5 || GIFLIB_MAJOR == 5 && GIFLIB_MINOR >= 1
 #define DGifCloseFile(gif) DGifCloseFile(gif, NULL)
