--- simage/CMakeLists.txt.orig	2020-11-30 20:05:24.851471101 +0100
+++ simage/CMakeLists.txt	2020-11-30 20:46:46.764692090 +0100
@@ -478,6 +478,10 @@
   target_link_libraries(simage PRIVATE ${VFW_LIBRARIES})
 endif()
 
+if(UNIX)
+  target_link_libraries(simage PRIVATE m)
+endif()
+
 # ############################################################################
 # Build examples
 # ############################################################################
